infra/modules/service/load_balancer.tf

```
@@ -82,6 +82,9 @@ resource "aws_lb_listener" "alb_listener_https" {
   port              = 443
   protocol          = "HTTPS"
   certificate_arn   = var.certificate_arn
+  mutual_authentication {
+    mode = module.app_config.app_name == "api" ? "passthrough" : "off"
```

```suggestion
    mode = startswith(var.service_name, "api-") ? "passthrough" : "off"
```

There's a context and variable scoping issue here, in that `module.app_config` won't be available at this location. You can access `var.service_name` though.

---

> There's a context and variable scoping issue here, in that module.app_config won't be available at this location.

I was worried about this, when nothing else in this folder referenced module. Made it two commits to make it easy to rollback.

> would be to add a enable_mutual_authentication variable that's false by default but the API sets it to true

Passthrough is just one of two ways you can configure the client auth and they require different additional config, so it's almost more like a whole object we might want to pass rather than the true/false? Maybe?